packages/core-utils/src/time.js

```diff
@@ -34,11 +35,9 @@ export function getLongDateFormat(config) {
  * @returns {string} formatted text representation
  */
 export function formatDuration(seconds) {
-  const dur = moment.duration(seconds, "seconds");
-  let text = "";
-  if (dur.hours() > 0) text += `${dur.hours()} hr, `;
-  text += `${dur.minutes()} min`;
-  return text;
+  return formatDistance(0, 1000 * seconds, {
+    includeSeconds: false
+  });
```

Comment:
This is a breaking change since `date-fns#formatDistance` outputs "fuzzy" time such as "about 2 hours". I think we should keep the original more exact time output.

Reply:
fixed in 55e76870 by migrating to [formatDistanceStrict](https://date-fns.org/v2.23.0/docs/formatDistanceStrict)